### PR TITLE
AARA-511 Troubleshoot CALS acceptance tests in PreInt

### PR DESCRIPTION
**Pull request by nasar-cambria**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/AARA-511

#### Brief feature description
- Troubleshoot CALS acceptance tests in PreInt

NOTE: I have commented out test cases in this file and added comments on the top. Please review them and please let me know if we can clean this up more. Also, once this is reviewed and approved I'll remove or update test cases.

By cleaning up our test cases and removing the repeated one the RSPEC Testing is still at **90.21 %** and finished testing the whole app in **5mins 55 seconds** comparatively to **7min and 51 seconds**

#### Does this PR block/effect other stories?
- 

#### Do any commands needs to be run in order to run the project with this PR?
- NA

#### Code coverage percent diff as a result of this PR?
- NA

### CODE REVIEW

File: spec/acceptance/rfa_forms/rfa_01c_application_spec.rb [hunk 1]
```diff
@@ -118,10 +122,16 @@
...
+    select 'Sr', from: 'identified_children[0].name_suffix'
+    select 'TK', from: 'grade'
+    fill_in('name_of_school', with: 'School name', match: :prefer_exact) 
     click_button('Save Progress')
     expect(find_field('identified_children[0].date_of_birth').value).to eq '01/01/2000'
+    expect(find_field('name_of_school').value).to eq 'School name'
+    expect(find_field('grade').value).to eq '1'
+    expect(find_field('identified_children[0].name_suffix').value).to eq '6'
     expect(page).to have_button('Submit', disabled: false)
+    click_button('Save Progress')
```
Comment (anuroop15) [inline spec/acceptance/rfa_forms/rfa_01c_application_spec.rb]: why is disabled: true expect statement removed?? isn't the submit button to be disabled be the requirement for RFA 01 C. ??

File: spec/support/capybara.rb [hunk 2]
```diff
@@ -65,9 +65,9 @@ def registar_chrome(capybara_config, selenium_browser)
 
 remote_capabilities(Capybara)
 
-# wait increased to 5 seconds to avoid random failures.
+# wait increased to 10 seconds to avoid random failures.
 # default wait is 2 seconds.
-Capybara.default_max_wait_time = 5
+Capybara.default_max_wait_time = 10
```
[the remark this answers is not in the document]
Comment (anuroop15) [reply spec/support/capybara.rb]: Just checking if there is a standard wait time ?

[the remark this answers is not in the document]
Comment (nasar-cambria) [reply spec/support/capybara.rb]: I noticed sometime's it was taking more than 5 second to load another page. I can change it back to 5 and try again.

File: spec/acceptance/rfa_forms/rfa_01c_application_spec.rb (hunk 1 above)
Comment (nasar-cambria) [reply spec/acceptance/rfa_forms/rfa_01c_application_spec.rb]: As discussed yesterday there is a bug in C form which I reported. So, once that is resolved we can add that test back again. 

Here is the ticket: https://osi-cwds.atlassian.net/browse/AARA-512

File: spec/acceptance/rfa_forms/rfa_01c_application_spec.rb [hunk 3]
```diff
@@ -6,46 +6,50 @@
...
-    page.driver.browser.manage.window.resize_to 1200, 800
-    click_button 'Create RFA Application'
-    first_name = 'Kimberley'
-    last_name = 'RReily'
-    fill_in('applicants[0].first_name', with: first_name, match: :prefer_exact)
-    fill_in('applicants[0].middle_name', with: 'k', match: :prefer_exact)
-    fill_in('applicants[0].last_name', with: last_name, match: :prefer_exact)
-    find('#child_identifiedYes').click
-    find('#child_in_homeYes').click
+    # We are validating this test cases in submit enabled test case on line 67 and I can add expect
```
Comment (ratneshraval) [inline spec/acceptance/rfa_forms/rfa_01c_application_spec.rb]: Any reason to keep the commented out tests instead of removing them?

Comment (nasar-cambria) [reply spec/acceptance/rfa_forms/rfa_01c_application_spec.rb]: I wanted to remove once everyone read my comments on it. I'll remove it now :)

File: spec/acceptance/rfa_forms/rfa_01a_application_spec.rb [hunk 4]
```diff
@@ -135,153 +137,41 @@
...
-
-  scenario 'check for disabled save button when applicant names are empty', set_auth_header: true do
-    visit root_path
-    click_button 'Create RFA Application'
-    expect(page).to have_content 'Applicant 1 - Information'
-    expect(page).to have_button('Save Progress', disabled: true)
-    fill_in 'applicants[0].first_name', with: '  '
-    fill_in 'applicants[0].last_name', with: '  '
-    expect(page).to have_button('Save Progress', disabled: true)
+    # DevNote: this test case will be updated to include more fields as submit
```
Comment (anuroop15) [inline spec/acceptance/rfa_forms/rfa_01a_application_spec.rb]: was this test covered in jasmine?? if not please create a tech debt.

Comment (nasar-cambria) [reply spec/acceptance/rfa_forms/rfa_01a_application_spec.rb]: Done.

**Pull request merged**